array.go

```diff
@@ -1004,18 +1005,18 @@ func (a *Array) GetMetadata(key string) (Datatype, uint, interface{}, error) {
 
 	ret := C.tiledb_array_get_metadata(a.context.tiledbContext, a.tiledbArray, ckey, &cType, &cValueNum, &cvalue)
 	if ret != C.TILEDB_OK {
-		return 0, 0, nil, fmt.Errorf("Error getting metadata from array: %s, key: %s", a.context.LastError(), key)
+		return 0, 0, nil, fmt.Errorf("error getting metadata from array: %w, key: %s", a.context.LastError(), key)
 	}
 
 	valueNum := uint(cValueNum)
 	if valueNum == 0 {
-		return 0, 0, nil, fmt.Errorf("Error getting metadata from array, key: %s does not exist", key)
+		return 0, 0, nil, fmt.Errorf("error getting metadata from array, key: %s does not exist", key)
 	}
 
 	datatype := Datatype(cType)
 	value, err := datatype.GetValue(valueNum, cvalue)
 	if err != nil {
-		return 0, 0, nil, fmt.Errorf("%s, key: %s", err.Error(), key)
+		return 0, 0, nil, fmt.Errorf("%w, key: %s", err, key)
```

Comment:
Might as well make this error consistent with the ones above and below.
```suggestion
		return 0, 0, nil, fmt.Errorf("error getting metadata from array: %w, key: %s", err, key)
```